src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return isDone ? "[X]" : "[ ]";
```

Comment:
Can consider merging this statement into the `toString()` method since it is likely to be used only in the `toString()` and not elsewhere